pkg/net/httpclient.go

```diff
@@ -11,9 +11,9 @@ import (
 	"github.com/giantswarm/clustertest/pkg/logger"
 )
 
-// NewHttpClient returns an initialized HTTP Client that uses an external nameserver to help avoid negative caching
+// NewHTTPClient returns an initialized HTTP Client that uses an external nameserver to help avoid negative caching
 // and if detected will make use of any proxy found in the environment
-func NewHttpClient() *http.Client {
+func NewHTTPClient() *http.Client {
```

Comment:
There was some requirement renaming the function from golangci-lint, those might have an impact. I can also ignore it if needed